.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/**
```

Comment:
That's an IDE-specific rule. Why only this IDE though and not all the other ones / text editors ? I would say either all or none, my preferred option being none. @ens-bwalts

Reply:
The first commit I did with the "top level" git ignore did the ignore for a bunch of IDE, if I reckon well. But as well, I can remove it. really up to you guys, you are the maintainer. Also, having "all of them" may be overkill, if only we add the ones used across ensembl, that would be great :-)